CONSTITUTION.md

```diff
@@ -56,8 +56,40 @@ NOTE: Minutes can be useful for disseminating information to members who can’t
 
 Only registered UCSD students may hold office in the organization. Only registered UCSD students may vote in elections for the selection of the organization’s officers.
 
-[Insert information about officer positions and selection process here]
-
+### Officers
```

Comment:
Feel free to leave single comments on specific parts of the file, like this comment!

to do this, hover on the left edge of the text, a blue ➕ will appear, and clicking that allows you to comment anywhere on the file!